lib/src/source/hosted.dart

```diff
@@ -117,8 +117,15 @@ class HostedSource extends Source {
     // Clearly, a bit of investigation is necessary before we update this to
     // pub.dev, it might be attractive to do next time we change the server API.
     try {
+      var defaultHostedUrl = 'https://pub.dartlang.org';
```

Comment:
This seems the type of thing that should be referenced from a shared const 🤷

Reply:
The `HostedSource.defaultUrl` while not const is the shared location for this information.